src/NuGet.Clients/NuGet.CommandLine/Commands/ProjectFactory.cs

```diff
@@ -143,25 +143,14 @@ private void Initialize(dynamic project)
             }
 
             // This happens before we obtain warning properties, so this Logger is still IConsole.
-            IConsole console = Logger as IConsole;
-            switch (LogLevel)
+            var console = (IConsole) Logger;
+            console.Verbosity = LogLevel switch
             {
-                case LogLevel.Verbose:
-                    {
-                        console.Verbosity = Verbosity.Detailed;
-                        break;
-                    }
-                case LogLevel.Information:
-                    {
-                        console.Verbosity = Verbosity.Normal;
-                        break;
-                    }
-                case LogLevel.Minimal:
-                    {
-                        console.Verbosity = Verbosity.Quiet;
-                        break;
-                    }
-            }
+                LogLevel.Verbose => Verbosity.Detailed,
+                LogLevel.Information => Verbosity.Normal,
+                LogLevel.Minimal => Verbosity.Quiet,
+                _ => console.Verbosity
+            };
```

Comment:
Please fix formatting
```suggestion
            var console = (IConsole)Logger;
            console.Verbosity = LogLevel switch
            {
                LogLevel.Verbose => Verbosity.Detailed,
                LogLevel.Information => Verbosity.Normal,
                LogLevel.Minimal => Verbosity.Quiet,
                _ => console.Verbosity
            };
```